hack/openshift/update-image-sha.sh

```diff
@@ -51,6 +51,7 @@ declare -A IMAGES=(
   ["s2i"]="registry.redhat.io/source-to-image/source-to-image-rhel8"
   ["ubi-minimal"]="registry.redhat.io/ubi8/ubi-minimal"
   ["java"]="registry.redhat.io/ubi8/openjdk-17"
+  ["oc"]="registry.redhat.io/openshift4/ose-cli"
```

Comment:
should we use the internal image that we get in image registry of cluster for this like the oc clustertask

Reply:
do you mean like this image in the oc clustertask?
image: image-registry.openshift-image-registry.svc:5000/openshift/cli:$(params.VERSION)

Reply:
we can use internal image registry image `image-registry.openshift-image-registry.svc:5000/openshift/cli:latest`